GNUmakefile

```diff
@@ -150,12 +150,14 @@ cov:
 	gocov test $(GOFILES) | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test: other-consul dev-build vet test-internal
+test: other-consul dev-build vet test-install-deps test-internal
+
+test-install-deps:
```

Comment:
Shouldn't this be added to the `.PHONY` variable. Just in case a file names test-install-deps does come to exist in our directory?

Reply:
Yep, good catch.